server/api/src/lib/clipAndCreate.ts

```
@@ -0,0 +1,208 @@
+  } catch {
+    return relative;
+  }
+}
+
+function cleanupHtml(html: string, doc: Document): string {
+  const div = doc.createElement("div");
+  div.innerHTML = html;
+
+  const unwanted = ["script", "style", "noscript", "iframe", "object", "embed", "form"];
+  for (const sel of unwanted) {
+    div.querySelectorAll(sel).forEach((el) => el.remove());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)